dev.sh

```diff
@@ -15,7 +15,7 @@ _start_gunicorn() {
   if [[ "$STATSD_HOST" ]]; then
     suffix="--statsd-host ${STATSD_HOST}:${STATSD_PORT}"
   fi
-  if [[ "$RUN_ENV" == "enterprise" ]] || [[ "$RUN_ENV" == "DEV" ]]; then
+  if [[ "$RUN_ENV" == "ENTERPRISE" ]] || [[ "$RUN_ENV" == "DEV" ]]; then
```

Comment:
I would make this either case insensitive compare or add another if. Just in case users are already accommodating for this being lower case.